package.json

```diff
@@ -22,6 +22,7 @@
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
     "axios": "^0.21.1",
+    "commitlint": "^13.1.0",
```

Comment:
Why we need this?
If it's supposed to be used, we need to move it to `devDependencies` otherwise should be removed.